src/components/topic/related/related-as-cards.js

```diff
@@ -0,0 +1,142 @@
+import { shortenString } from '../../../utils/string'
+import base from './base'
+import Image from '@twreporter/react-article-components/lib/components/img-with-placeholder'
+import mq from '../../../utils/media-query'
+import React from 'react'
+import styled from 'styled-components'
+
+const ImageBorder = styled.div`
+  ${mq.tabletAndAbove`
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    border: 0 solid #ffffff;
+    transition: border .1s ease;
+  `}
+`
+
+const ItemsContainer = styled(base.ItemsContainer)`
+  ${mq.tabletAndAbove`
+    padding: 60px 0 50px 0;
+    display: flex;
+    margin: 0 auto;
+    background-color: inherit;
+    width: 90%;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: flex-start;
+  `}
+`
+
+const ItemLink = styled(base.ItemLink)`
+  ${mq.tabletAndAbove`
+    flex: 0 0 300px;
+    width: 300px;
+    margin: 0 10px 50px 10px;
+    box-shadow: none;
+    transition: top .1s ease, box-shadow .2s ease, transform .1s ease;
+    &::before {
+      transition: border-width .1s ease-out;
+      content: "";
+      width: 0;
+      height: 0;
+      top: -2px;
+      left: -2px;
+      position: absolute;
+      z-index: 5;
+      border-style: solid;
+      border-width: 25px 25px 0 0;
+      border-color: #fff transparent transparent;
+    }
+    :hover {
+      &::before {
+        border-width: 0;
+      }
+      transform: translateY(-5px);
+      box-shadow: 0 5px 15px 0 rgba(0,0,0,.25);
+      ${base.ItemMeta} {
+        transform: translateY(0);
+      }
+      ${ImageBorder} {
+        border: 5px solid #ffffff;
+      }
+      ${base.ItemImageSizing}, ${base.ItemMeta} {
+        box-shadow: none;
+      }
+    }
+  `}
+`
+
+
+const ItemImageSizing = styled(base.ItemImageSizing)`
+  ${mq.tabletAndAbove`
+    width: 100%;
+    height: 200px;
+    position: relative;
+    box-shadow: 0 2px 5px 0 rgba(0,0,0,.1);
+    transition: box-shadow .1s ease, transform .2s ease;
+  `}
+`
+
+const ItemMeta = styled(base.ItemMeta)`
+  ${mq.tabletAndAbove`
+    width: 100%;
+    min-height: 222px;
```

Comment:
Since the posts in English do not looks well with `text-align: justify`, I did not adherent the previous code that truncate the text according to height, but according to letters count.

In the most cases, the description text will not extend the card because the description has been truncated.

But there may be rare cases that the content extend the height due to the word break or word wrap rule. (When there are less letters in a line than expected).

Reply:
We can apply `display: flex; flex-direction: column;` to the card container to make the cards in exception case shared the same height.